packages/download-url/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "mongodb-download-url",
+  "version": "1.5.4",
+  "description": "Lookup download URLs for MongoDB versions.",
+  "scripts": {
+    "lint": "eslint src/**/*.ts test/**/*.ts",
```

Comment:
I think we'd want a `check` script here as well, depcheck, etc.

Typically the approach I've taken for moving packages is to create a new one using the `create-workspace` script, moving over the `.ts` files from the old repo, other files as needed (e.g. LICENSE), and manually adjusting the package.json – it's more work, but it helps ensure a more consistent experience between the packages in the monorepo here

Reply:
Makes sense. I tried doing that and added a commit breakdown to clarify what's happening since now we have actual code changes and is not a pure transfer. LMK if that makes sense to you.